packages/react-app/src/components/common/ConnectWeb3.jsx

```diff
@@ -73,7 +74,26 @@ export const ConnectWeb3 = () => {
         </Badge>
       </Tooltip>
     ) : (
-      networkName
+      <span style={{ fontWeight: 'bold' }}>{networkName}</span>
+    );
+  };
+
+  const renderHelperBox = () => {
+    const { chainId, status } = isChainIdInjected;
+    if (status && [HOME_CHAIN_ID, FOREIGN_CHAIN_ID].includes(chainId)) {
+      return (
+        <Text color="greyText" mb={4} textAlign="center">
```

Comment:
Here we should also change the title from `Switch to supported network` to perhaps `Switch to chosen network`

Reply:
Yup. Noob in English lol.